pages/common/npm.md

```diff
@@ -10,6 +10,10 @@
 
 `npm install`
 
+- Download and install a module globally so it's always available in the shell
```

Comment:
This isn't always true, you could spawn a node shell with a different NODE_LIB path that knows nothing about the global npm modules. Also it could be interpreted as available from the terminal shell (bash, csh, zsh) and thus might be misleading – most packages can be installed globally and don't provide any binaries.